ginivision/src/main/java/net/gini/android/vision/review/ReviewActivity.java

```diff
@@ -247,11 +272,25 @@ private void showFragment() {
 
     @Override
     public void onProceedToAnalysisScreen(@NonNull Document document) {
-        mAnalyzeDocumentActivityIntent.putExtra(AnalysisActivity.EXTRA_IN_DOCUMENT, document);
-        if (mDocumentAnalysisErrorMessage != null) {
-            mAnalyzeDocumentActivityIntent.putExtra(AnalysisActivity.EXTRA_IN_DOCUMENT_ANALYSIS_ERROR_MESSAGE, mDocumentAnalysisErrorMessage);
+        if (mNoExtractionsFound) {
```

Comment:
If we were told, that no extractions were found we can check whether we should show the No Results Screen. If we shouldn't we simply finish and return to the Camera Screen. If the No Results Screen is shown we also finish the Review Screen so that when the No Results Screen is closed we end up back at the Camera Screen.